psalm.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<psalm
+    errorLevel="6"
```

Comment:
are errors occurring at lower values?

Reply:
Documentation [says](https://psalm.dev/docs/running_psalm/error_levels/): Level 1 is the most strict, level 8 is the most lenient.